src/frontend/src/common/components/library/data_table/style.ts

```diff
@@ -73,3 +84,20 @@ export const HeaderCheckbox = styled(Checkbox)`
     `;
   }}
 `;
+
+export interface AlignProps extends Props {
+  align?: string;
+}
+
+export const TableHeader = styled("div", {
+  shouldForwardProp: (prop) => !doNotForwardProps.includes(prop as string),
+})`
+  ${(props: AlignProps) => {
+    const { align } = props;
+    const justify = align !== undefined ? align : "left";
```

Comment:
```suggestion
    const justify = align ? align : "left";
```

Reply:
This is the TS way of saying "set the value to variable, but if it doesn't exist use this value instead"